routes/users.route.js

```diff
@@ -369,21 +384,37 @@ router.post("/", async(req, res, next) => {
         res.send(msg)
     } else if (data[0] == '2' && data[1] == '3' && data[2] !== '' && data[3] !== '' && data[4] !== '' && data[5] !== '' && data[6] == '2' && data[7] !== '' && data[8] !== '' && data[9] == null) {
         sellerMSISDN = phoneNumber
-        escrowMSISDN = phoneNumber // Todo: change to nigeria phone code '+234'
-        agentMSISDN = phoneNumber  // Todo: add agent phone number
+        escrowMSISDN = '+254712380880'  // Todo: change to nigeria phone code '+234' escrowMSISDN
+        agentMSISDN = '+254712380880'  // Todo: add agent phone number
 
         coinToSell = data[2]
         amountToSell = data[3]
         cashPickupLocation = data[4]
-        lga = data[5]
+        localGovernmentArea = data[5]
         pickupPerson = data[6]
         recipientMSISDN = data[7]
         userInputPin = data[8]
 
         en_userInputPin = encryptionPin(userInputPin.toString())
         verificationId = generateVerificationId()
 
-        console.log('coinToSell', coinToSell, 'amountToSell', amountToSell, 'cashPickupLocation', cashPickupLocation, 'lga', lga, 'pickupPerson', pickupPerson, 'userInputPin', userInputPin, 'recipientMSISDN', recipientMSISDN)
+        // cashPickupLocation
+        if(data[4]==='1'){cashPickupLocation = 'Ikeja'}
```

Comment:
402 --> 417  what is  happening with the other location option.

Reply:
Let me check on my side  if the same is happening

Reply:
the user is supposed to select the location they want to pick up their cash